ipu-plugin/pkg/ipuplugin/deviceplugin.go

```diff
@@ -8,27 +8,30 @@ import (
 	"slices"
 	"strings"
 
+	"github.com/intel/ipu-opi-plugins/ipu-plugin/pkg/types"
 	pb "github.com/openshift/dpu-operator/dpu-api/gen"
 	pluginapi "k8s.io/kubelet/pkg/apis/deviceplugin/v1beta1"
 )
 
 type DevicePluginService struct {
 	pb.UnimplementedDeviceServiceServer
+	mode string
```

Comment:
Maybe a comment pointing to the type enum and valid choices ?

Reply:
this one should be easy to lookup. For now, sticking with the convention in this repo...it doesnt have comments for types.